funcbench/main.go

```diff
@@ -99,35 +124,46 @@ The default is 2 hours (2h).`).Default("2h").Duration()
 				err error
 			)
 
-			if *gitHubPRNumber == 0 {
-				env, err = newLocalEnv(environment{
-					logger:        logger,
-					benchFunc:     *benchFunc,
-					compareTarget: *compareTarget,
-					home:          os.Getenv("HOME"),
-				})
+			// Setup Environment.
+			e := environment{
+				logger:        logger,
+				benchFunc:     cfg.benchFuncRegex,
+				compareTarget: cfg.compareTarget,
+			}
+			if cfg.ghPr == 0 {
+				// Local Mode.
+				env, err = newLocalEnv(e)
 				if err != nil {
-					return errors.Wrap(err, "new env")
+					return errors.Wrap(err, "environment creation error")
```

Comment:
We are in errors.Wrap... what else it than error? (: 
```suggestion
					return errors.Wrap(err, "environment create")
```

Reply:
It's our opinion, so feel free to say no.

It's based on this we just are about to share: https://github.com/thanos-io/thanos/blob/389197775d5008fb8f52b1b3f9bd95da8bda9e1f/docs/contributing/coding-style-guide.md